sample/src/main/kotlin/app/futured/donutsample/ui/playground/PlaygroundActivity.kt

```diff
@@ -1,23 +1,23 @@
-package com.thefuntasty.donutsample.ui.playground
+package app.futured.donutsample.ui.playground
 
 import android.os.Bundle
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/17ecfffa79363fc193b8c29569ad5912bbffe3e2/sample/src/main/kotlin/app/futured/donutsample/ui/playground/PlaygroundActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>